src/Pact/Types/Parser.hs

```diff
@@ -45,3 +64,87 @@ style = IdentifierStyle "atom"
         (HS.fromList ["true","false"])
         Symbol
         ReservedIdentifier
+
+-- -------------------------------------------------------------------------- --
+-- Pact Attoparsec backend parser
+
+-- | A wrapper around Attoparsec that adds DeltaParsing
+--
+newtype PactAttoparsec a = PactAttoparsec
+    { runPactAttoparsec :: StateT Int AP.Parser a }
+    deriving (Functor, Applicative, Alternative, Monad, MonadPlus, MonadState Int)
+    deriving (Parsing, TokenParsing)
+
+pactAttoParseOnly :: PactAttoparsec a -> T.Text -> Either String a
+pactAttoParseOnly = AP.parseOnly . flip evalStateT 0 . runPactAttoparsec
+
+-- | Atto DeltaParsing instance provides 'position' only (with no support for
+-- hidden chars like Trifecta).
+--
+instance DeltaParsing PactAttoparsec where
+    line = return mempty
```

Comment:
What about something like this:
```suggestion
    line = PactAttoparsec (fail "DeltaParsing.line is not supported for PactAttoparsec")
```
The error message is more informative, than giving an empty string.

Reply:
assuming it's actually not used ...